h/static/styles/annotation.scss

```diff
@@ -42,12 +70,14 @@ $annotation-card-left-padding: 10px;
 .annotation-header {
   display: flex;
   flex-direction: row;
-  align-items: flex-start;
-  margin-bottom: 5px;
+  align-items: baseline;
+  // Align the baseline of the text with the bottom of a containing block element.
+  margin-top: -$layout-h-margin + 10px;
```

Comment:
This doesn't do what the comment above says - it is setting the top margin not the bottom margin. What might be better here is for the comment to explain the desired spacing from a visual point of view. eg:

`// Margin between x-height of username and top of annotation card should be ~Npx`

That way if we think of a better way of achieving this in future it'll be clear what we are aiming for.